scalyr_agent/agent_main.py

```diff
@@ -1257,18 +1265,25 @@ def __create_client(self, quiet=False):
         @rtype: ScalyrClientSession
         """
         if self.__config.verify_server_certificate:
+            is_dev_install = INSTALL_TYPE == DEV_INSTALL
+            is_dev_or_msi_install = INSTALL_TYPE in [DEV_INSTALL, MSI_INSTALL]
+
             ca_file = self.__config.ca_cert_path
             intermediate_certs_file = self.__config.intermediate_certs_path
 
             # Validate provided CA cert file and intermediate cert file exists. If they don't
             # exist, throw and fail early and loudly
-            if not os.path.isfile(ca_file):
+            if not is_dev_install and not os.path.isfile(ca_file):
                 raise ValueError(
                     'Invalid path "%s" specified for the "ca_cert_path" config '
                     "option: file does not exist" % (ca_file)
                 )
 
-            if not os.path.isfile(intermediate_certs_file):
+            # NOTE: We don't include intermediate certs in the Windows binary so we skip taht chek
```

Comment:
s/taht/that/

Reply:
I wish there was a "steven spell check" plugin available for vim, it seems to be more effective than built in vim spellcheck plugin :D